com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -442,7 +477,7 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                     }
                     else
                     {
-                        m_DepthNormalPrepass.Setup(cameraTargetDescriptor, m_DepthTexture, m_NormalsTexture);
+                        m_DepthNormalPrepass.Setup(cameraTargetDescriptor, m_DepthTexture, m_NormalsTexture, useDepthPriming);
```

Comment:
No need to pass this as argument as the pass already has access to the renderer.

Reply:
Indeed. Initially kept this in because I setup the normal descriptors in the setup based on depth priming being enabled or not but it's possible to alter the description as needed during OnCameraSetup(). Fixing.